index.js

```diff
@@ -31,7 +31,7 @@ module.exports = options => {
         let config = getConfig(options);
       }
       return new Promise((resolve, reject) => {
-        if (config.startOnInitialise === false) {
+        if (config.start === false) {
```

Comment:
the config is not being generated from the defaults when start is called independently from the initial call to bootstrap